docs/guides/mariadb/backup/kubestash/logical/index.md

```diff
@@ -0,0 +1,704 @@
+
+Verify that the `CronJob` has been created using the following command,
+
+```bash
+$ kubectl get cronjob -n demo
+NAME                                             SCHEDULE     TIMEZONE   SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+trigger-sample-mariadb-backup-frequent-backup   */5 * * * *   <none>     False     0        <none>          4m23s
+```
+
+**Verify BackupSession:**
+
+KubeStash triggers an instant backup as soon as the `BackupConfiguration` is ready. After that, backups are scheduled according to the specified schedule.
+
+```bash
+$ kubectl get backupsession -n demo -w
+NAME                                                INVOKER-TYPE          INVOKER-NAME              PHASE       DURATION   AGE
+sample-mariadb-backup-frequent-backup-1725449400    BackupConfiguration   sample-mariadb-backup     Succeeded              7m22s
+```
+
+We can see from the above output that the backup session has succeeded. Now, we are going to verify whether the backup data has been stored in the backend.
+
+**Verify Backup:**
+
+Once a backup is complete, KubeStash will update the respective `Repository` CR to reflect the backup. Check that the repository `sample-mariadb-backup` has been updated by the following command,
+
+```bash
+$ kubectl get repository -n demo gcs-mariadb-repo
+NAME                       INTEGRITY   SNAPSHOT-COUNT   SIZE    PHASE   LAST-SUCCESSFUL-BACKUP   AGE
+gcs-mariadb-repo           true        1                806 B   Ready   8m27s                    9m18s
+```
+
+At this moment we have one `Snapshot`. Run the following command to check the respective `Snapshot` which represents the state of a backup run for an application.
+
+```bash
+$ kubectl get snapshot.storage.kubestash.com -n demo -l=kubestash.com/repo-name=gcs-mariadb-repo
+NAME                                                                  REPOSITORY          SESSION           SNAPSHOT-TIME          DELETION-POLICY   PHASE       AGE
+gcs-mariadb-repo-sample-mariadb-ckup-frequent-backup-1726569774       gcs-mariadb-repo    frequent-backup   2024-09-17T10:43:04Z   Delete            Succeeded   41m
+```
+
+> Note: KubeStash creates a `Snapshot` with the following labels:
+> - `kubestash.com/app-ref-kind: <target-kind>`
+> - `kubestash.com/app-ref-name: <target-name>`
+> - `kubestash.com/app-ref-namespace: <target-namespace>`
+> - `kubestash.com/repo-name: <repository-name>`
+>
+> These labels can be used to watch only the `Snapshot`s related to our target Database or `Repository`.
+
+If we check the YAML of the `Snapshot`, we can find the information about the backup components of the Database.
+
+```bash
+$ kubectl get snapshot.storage.kubestash.com -n demo gcs-mariadb-repo-sample-mariadb-ckup-frequent-backup-1726569774 -oyaml
+```
+
+```yaml
+apiVersion: storage.kubestash.com/v1alpha1
+kind: Snapshot
+metadata:
+  creationTimestamp: "2024-09-17T10:43:04Z"
+  finalizers:
+    - kubestash.com/cleanup
+  generation: 1
+  labels:
+    kubedb.com/db-version: 11.1.3
```

Comment:
`kubedb.com/db-version: 11.1.3` This should be in annotations. Change all the places